docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,107 @@
+# Release 381 (16 May 2022)
+
+## General
+
+* Add support for fault-tolerant execution with exchange spooling on Azure Blob Storage. ({issue}`12211`)
+* Add support for table functions. ({issue}`1839`)
+* Increase the default number of stages allowed for a query from 100 to 150,
+  specified with `query.max-stage-count`, so large queries do not fail. ({issue}`12292`)
+* Allow configuring the number of partitions for distributed joins and
+  aggregations executed with fault-tolerant execution enabled
+  (`retry-policy=TASK`) with the `fault-tolerant-execution-partition-count`
+  configuration property or the `fault_tolerant_execution_partition_count`
+  session property. ({issue}`12263`)
+* Rename configuration property used for selecting a policy for low memory query
```

Comment:
This one should no longer be needed

Reply:
Hahahah ... too funny.. .rename without rename...

Reply:
Removed..